package/samplers/catcma/README.md

```diff
@@ -7,6 +7,13 @@ optuna_versions: [3.6.1]
 license: MIT License
 ---
 
+## Abstract
+
+The cutting-edge evolutionary computation algorithm CatCMA has been published on OptunaHub.
+CatCMA is an algorithm that excels in mixed search spaces with continuous and discrete variables.
+
+![The performance comparison results of CatCMA and other algorithms](images/catcma-performance.png)
```

Comment:
I guess this figure is excerpted from the paper. Could you show the reference, please?

Reply:
I add the reference. Thanks for the comment.